src/client.spec.ts

```diff
@@ -1390,36 +1390,46 @@ describe('ReactSDKClient', () => {
       expect(mockFn).toHaveBeenCalledTimes(0);
     });
 
-    it('track works as expected', () => {
-      const mockFn = mockInnerClient.track as jest.Mock;
-
-      instance.track('evt1');
-
-      expect(mockFn).toHaveBeenCalledTimes(1);
-      expect(mockFn).toHaveBeenCalledWith('evt1', 'user1', { foo: 'bar' }, undefined);
+    describe('track with different parameters', () => {
+      it('track with only event key, calls inner client with valid arguments', () => {
+        const mockFn = mockInnerClient.track as jest.Mock;
+        instance.track('evt1');
 
-      mockFn.mockReset();
+        expect(mockFn).toHaveBeenCalledTimes(1);
+        expect(mockFn).toHaveBeenCalledWith('evt1', 'user1', { foo: 'bar' }, undefined);
+      });
 
-      instance.track('evt1', 'user2', { bar: 'baz' });
+      it('track with event key and overrided user id and attributes, calls inner client with valid arguments', () => {
+        const mockFn = mockInnerClient.track as jest.Mock;
+        instance.track('evt1', 'user2', { bar: 'baz' });
 
-      expect(mockFn).toHaveBeenCalledTimes(1);
-      expect(mockFn).toHaveBeenCalledWith('evt1', 'user2', { bar: 'baz' }, undefined);
+        expect(mockFn).toHaveBeenCalledTimes(1);
+        expect(mockFn).toHaveBeenCalledWith('evt1', 'user2', { bar: 'baz' }, undefined);
+      });
 
-      mockFn.mockReset();
+      it('track with event key and event tags, calls inner client with valid arguments', () => {
+        const mockFn = mockInnerClient.track as jest.Mock;
+        instance.track('evt1', { tagKey: 'tagVal' });
```

Comment:
Would it be good to explicitly pass `undefined` as the second parameter or use named parameters here and the last test of this `describe` block?

Reply:
`_client=null` is already covered in the previous block